tutorials/cloud_access/euclid-cloud-access.md

```diff
@@ -104,19 +104,13 @@ coord = SkyCoord.from_name(target_name)
 search_radius = 10 * u.arcsec
 ```
 
-List all Simple Image Access (SIA) collections for IRSA:
+List all Simple Image Access (SIA) collections for IRSA with names containing "euclid":
 
 ```{code-cell} ipython3
-collections = Irsa.list_collections(servicetype='SIA')
+collections = Irsa.list_collections(servicetype='SIA', filter='euclid')
 len(collections)
```

Comment:
I was printing length here and then displayed filtered results in the next cell. To maintain the narrative "As per ... above table...", we should show results here:

```suggestion
collections
```